javascript/src/main/pages/Home/Home.js

```diff
@@ -1,18 +1,37 @@
 import React from "react";
-import { Jumbotron } from "react-bootstrap";
 import {useAuth0} from "@auth0/auth0-react";
+import { CardDeck, Card } from "react-bootstrap";
 
 const Home = () => {
     const { isAuthenticated: _isAuthenticated } = useAuth0();
 
     return (
-            <Jumbotron>
-                <div className="text-left">
-                    <h5>Welcome to the Mapache Search!</h5>
-                    <p>This is where your home page content goes
-                    </p>
-                </div>
-            </Jumbotron>
+            <div>
```

Comment:
Minor style issue, can you align this tag?

Reply:
This still needs to be done before this is mergable.  This tag does need to be aligned.